test/performance-tests.sh

```diff
@@ -24,6 +24,18 @@ function teardown() {
   uninstall_knative_serving
 }
 
```

Comment:
I suggest adding before the function name

`# On a Prow job, save some metadata about the test for Testgrid.`

and after that

`  (( ! IS_PROW )) && return`

so you don't have local errors with the test trying to write a file to the root dir. :)